### DISCUSSION TITLE
can't find product key from Unattended file

### DISCUSSION THREAD

**Discussion by halinwu** [vmware/packer-examples-for-vsphere#894] [Help] [answered]
### Code of Conduct

- [X] I have read and agree to the project's Code of Conduct.

### Project Version

develop

### VMware vSphere

8.0.2

### HashiCorp Packer

1.10.2

### HashiCorp Packer Plugin for VMware vSphere

1.2.7

### Guest Operating System

CentOS 9 

### Description

I download the latest file  yesterday,  I wanted to deploy windows OS , but I encountered a issue, even I added the product key at "windows-server-20**.pkrvars.hcl" or "data/autounattend.pkrtpl.hcl" , it showed the same issue. and I deployed windows 11, windows 2019 and windows 2022,  it cloudn't work.  I'm very confuse , I deployed those template at version 23.11 succesfully monthes ago,  I founded the some diffrent between the 2 version, please tell me what neccesary operations could  fix the issue ? 

### Expected Behavior

deploye os automatically

### Actual Behavior

can't find product key from unattend file

### Steps to Reproduce

1. git clone ;
2. modify common , vsphere, build etc HCL files; 
3. config.sh 
4.excute  build.sh to create template , it colud create vm and  load os, but stuck product key , show me can't found product key;

**Comment by tenthirtyam** [comment]
Please refer to the CHANGELOG for v0.19.0 where Windows evaluation is used by default. You may need to update your variables based on the latest.

**Comment by halinwu** [comment, marked as answer]
thanks a lot !

**Reply by tenthirtyam** [reply]
https://github.com/vmware-samples/packer-examples-for-vsphere/issues/896
